apps/src/templates/sectionsRefresh/coteacherSettings/AddCoteacher.jsx

```diff
@@ -34,10 +34,10 @@ export const getInputErrorMessage = (email, coteachersToAdd, sectionId) => {
     if (response.ok) {
       return '';
     }
-    if (response.errorThrown === 'Not Found') {
+    if (response.status === 404 && response.statusText === 'Not Found') {
       return i18n.coteacherAddNoAccount({email});
     }
-    if (response.errorThrown === 'Forbidden') {
+    if (response.status === 404 && response.statusText === 'Forbidden') {
```

Comment:
Shouldn't the status code be 403 for Forbidden?

Reply:
Yep! Fixed.